_posts/2016-06-17-using-xhprof.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Using XHProf to profile your Drupal module"
+date: 2016-06-17
+author: Tim Stallmann
+tags: php performance drupal drupal-planet module-development
+summary: A case study of using XHProf to profile a Drupal module.
+drupal_planet_summary:
+  Second part in a series of how to use XHProf effectively within a VM for a Drupal website.
+---
+
+After you've [installed XHProf](/2016/05/26/installing-xhprof.html), what's next? How can you make use of its recommendations to tune a module or a website? Unfortunately there are no hard-and-fast rules for optimizing code to run more efficiently. And I'm no expert on the matter, either. What I can offer here is my own experience trying to optimize a D8 module using XHProf.
```

Comment:
I would reference that it's part 2 somehow more formally from the get go and link to part 1. See http://savaslabs.com/2016/02/24/drupal-6-part-4.html